Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/storage/CoronaTestStorage.kt

```diff
@@ -0,0 +1,23 @@
+package de.rki.coronawarnapp.coronatest.storage
+
+import de.rki.coronawarnapp.coronatest.type.CoronaTest
+import timber.log.Timber
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class CoronaTestStorage @Inject constructor() {
+
+    suspend fun load(): Set<CoronaTest> {
+        Timber.tag(TAG).d("load()")
+        throw NotImplementedError()
+    }
+
+    suspend fun save(tests: Set<CoronaTest>) {
```

Comment:
Do you always save a set of test?, I think it should be for one test

Reply:
It's always a set, it's a snapshot of the current tests. The tests will live inside a flow  while the app is running,  via `HotDataFlow`, always in memory. On data change the snapshot will be updated via `save` and when the app was killed, they will be restored via `load`.